src/extensions/scenario/scenarioext.cpp

```diff
@@ -196,6 +200,9 @@ bool ScenarioClassExtension::Read_INI(CCINIClass &ini)
 {
     //EXT_DEBUG_TRACE("ScenarioClassExtension::Read_INI - 0x%08X\n", (uintptr_t)(This()));
 
+    const char* const BASIC = "Basic";
```

Comment:
This should be static, so;
`static const char * const BASIC = "Basic";` _(not the formatting of the `*` again)_.
Also, add a blank line between its definition and the INI operation please.